docs/api/locales.md

```diff
@@ -1,4 +1,6 @@
-# Possible Values of `app.getLocale()`
+# Locales
+
+> Locale values returned by `app.getLocale()`.
```

Comment:
Maybe this should be `Possible locale values`, currently it seems to suggest that all of these values are returned by `app.getLocale()` instead of just one.